ui/components/app/currency-input/currency-input.js

```diff
@@ -150,14 +154,19 @@ export default function CurrencyInput({
       suffix={shouldUseFiat ? secondarySuffix : primarySuffix}
       onChange={handleChange}
       value={initialDecimalValue}
+      className={className}
```

Comment:
The `classnames` documentation suggests falsy values are ignored, so I don't see an `undefined` class when omitted.  But perhaps "" is safer in case its used directly and not through a `classnames`

Reply:
Will also help when we eventually migrate this to TypeScript